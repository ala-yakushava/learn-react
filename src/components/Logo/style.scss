@import "../../styles/theme";

.Logo {
  margin: 0;
  font-size: 24px;
  color: $red;

  b {
    font-weight: 700;
  }
}
