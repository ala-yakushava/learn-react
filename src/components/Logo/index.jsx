import React from 'react';

import './style.scss';

const Logo = () => {
  return (
    <p className="Logo">
      <b>netflix</b>roulette
    </p>
  );
}

export default Logo;
