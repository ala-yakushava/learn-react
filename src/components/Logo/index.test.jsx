import React from 'react';
import renderer from 'react-test-renderer';

import Logo from './index';

describe('Components - Logo', () => {
  test('should match snapshot', () => {
    const tree = renderer
      .create(<Logo />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
