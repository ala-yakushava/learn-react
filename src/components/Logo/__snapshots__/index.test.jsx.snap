// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Logo should match snapshot 1`] = `
<p
  className="Logo"
>
  <b>
    netflix
  </b>
  roulette
</p>
`;
